1436.destination-city.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * @param {string[][]} paths
+ * @return {string}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/destination-city/
+*/
+
+var destCity = function(paths) {
+    const pathTable = {};
```

Comment:
```suggestion
    const pathTable = new Map();
```

Reply:
I updated the initialization of pathTable as you described above.